### ISSUE TITLE
Use Symfony style for Constraint (Assert) annotations 

### ISSUE THREAD

**Issue by annuh** [rectorphp/rector#6568]
# Question

Since https://github.com/rectorphp/rector-symfony/pull/16 Rector can automatically update our validation annotations to attributes. 👍 

```diff
-     * @Assert\Email(message="Invalid e-mail address")
+    #[\Symfony\Component\Validator\Constraints\Email(message: 'Invalid e-mail address')]
```

This is great, but unfortunately it doesn't use [Symfony's conventions](https://symfony.com/doc/current/validation.html):
```php
use Symfony\Component\Validator\Constraints as Assert;

    #[Assert\Email(message: 'Invalid e-mail address')]]
```

Is this possible using Rector, or is there some codesniffer rule/fixer we can use for this?

I found something related, but it only works for annotations: https://github.com/rectorphp/rector/blob/main/rules/Restoration/Rector/Namespace_/CompleteImportForPartialAnnotationRector.php

**Comment by TomasVotruba** [comment]
Thanks for reporting.

We talked about somethign similar in this issue:
https://github.com/rectorphp/rector/issues/6565#issuecomment-877820601

tl;dr; We don't support it for economical reasons, but would accept such feature.
